app/views/videos/show.html.haml

```diff
@@ -15,34 +15,27 @@
 %section.reviews.container
   .row
     .col-sm-10.col-sm-offset-1
-      %form
+      = bootstrap_form_for [ @video, @review ] do |f|
         %fieldset
           .form-group
-            %label Rate this video
             .row
               .col-sm-3
-                %select.form-control(name="")
-                  %option(value="5") 5 Stars
-                  %option(value="4") 4 Stars
-                  %option(value="3") 3 Stars
-                  %option(value="2") 2 Stars
-                  %option(value="1") 1 Star
+                = f.select :rating, select_collection_rating_values, { label: 'Rate this video' }
           .form-group
-            %label Write Review
             .row
               .col-sm-8
-                %textarea.form-control(name="" rows="6")
+                = f.text_area :content, { label: 'Write a review', rows: 6 }
```

Comment:
I think you do not have to pass other options as hash. `= f.text_area :content, label: 'Write a review', rows: 6` should work as well.